indico/modules/events/registration/client/js/form/fields/PhoneInput.jsx

```diff
@@ -8,15 +8,12 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
-export default function PhoneInput({htmlName, disabled}) {
-  return <input type="tel" name={htmlName} disabled={disabled} />;
```

Comment:
Why was the `disabled` logic removed? This is still needed - e.g. when submitting a registration we want to be able to disable the fields...

Reply:
I forgot to include it in the `<Form.Field>`, should be fixed

Reply:
It's needed in both places; the field just affects the display, but the inputs need to be disabled as well